docs/README.md

```diff
@@ -18,17 +13,6 @@ pip install -r requirements.txt
 make html
 ```
 
-## Links
+## Wiki
 
-* [Installation Guide](https://github.com/Microsoft/LightGBM/wiki/Installation-Guide)
-* [Quick Start](./Quick-Start.md)
-* [Python Quick Start](./Python-intro.md)
-* [Features](https://github.com/Microsoft/LightGBM/wiki/Features)
-* [Experiments](https://github.com/Microsoft/LightGBM/wiki/Experiments)
-* [Parameters](./Parameters.md)
-* [Parameters Tuning](./Parameters-tuning.md)
-* [Python API Reference](./Python-API.md)
-* [Parallel Learning Guide](https://github.com/Microsoft/LightGBM/wiki/Parallel-Learning-Guide)
-* [GPU Tutorial](./GPU-Tutorial.md)
-* [FAQ](./FAQ.md)
-* [Development Guide](./development.rst)
+In addition to our documentation hosted on Read the Docs, some additional topics are explained at https://github.com/Microsoft/LightGBM/wiki.
```

Comment:
it's ok to remove them.
(https://github.com/Microsoft/LightGBM/blob/master/docs/index.rst is imporant for the read the docs)